modules/analytics/src/views/full/style.scss

```diff
@@ -265,6 +265,15 @@ $success-color: #56b149;
       span:nth-child(2) {
         margin-left: 5px;
       }
+
+      &.disabled {
+        color: black;
```

Comment:
@MaxCloutier should I use another color for deleted QnAs ?

Reply:
seashell or shark

Reply:
Done in 38724ec . I ended up using `reef` as per Caro's recommendations.